src/environments/environments/CarBeatEnvironment.py

```diff
@@ -69,7 +69,6 @@ def __init__(self,
                  track='multi_track',
                  observation_mode='lidar_only',
                  max_goals=500,
-                 num_lidar_points=10
```

Comment:
Don't think this should be removed. 10 lidar points may help with driving but less sure for overtaking, plus it's removing functionality.

Reply:
Fixed in latest commit